tests/Uri/BuildTest.php

```diff
@@ -8,15 +8,23 @@ class BuildTest extends \PHPUnit\Framework\TestCase
 {
     /**
      * @dataProvider buildUriData
+     *
+     * @param string $value
      */
-    public function testBuild($value)
+    public function testBuild($value): void
     {
+        /** @var array<string, int|string> $parsedUrl */
+        $parsedUrl = parse_url($value);
+        $this->assertIsArray($parsedUrl);
```

Comment:
`parse_url()` can return `false` and other stuff when fed bad or non-standard input. `phpstan` complained about that possibility - we can only feed `$parsedUrl` to `build()` if it is an array.

So I made PHPdoc for the variable, and assert that it is an array.

That will make the test fall over if we feed it a test case that is already invalid input to `parse_url()`

Reply:
great change!

in this case we can see how great phpstan and phpunit integrates with each other <2